src/main/java/org/web3j/protocol/core/filters/BlockFilter.java

```diff
@@ -34,5 +36,10 @@ void process(List<EthLog.LogResult> logResults) {
             }
         }
     }
+
+    @Override
+    protected Request<?, EthLog> createFilterRequest(BigInteger filterId) {
+        return this.web3j.ethGetFilterChanges(filterId);
```

Comment:
It would be nice if this could be a no-op, with a comment that getFilterChanges is not supported for BlockFilters.